src/GeniusYield/Transaction.hs

```diff
@@ -105,37 +104,14 @@ data GYBuildTxEnv = GYBuildTxEnv
     , gyBTxEnvProtocolParams :: !(AlonzoCore.PParams (Api.S.ShelleyLedgerEra Api.S.BabbageEra))
     , gyBTxEnvPools          :: !(Set Api.S.PoolId)
     , gyBTxEnvOwnUtxos       :: !GYUTxOs
-    -- ^ own utxos available for use as additional input
+    -- ^ own utxos available for use as _additional_ input
     , gyBTxEnvChangeAddr     :: !GYAddress
     , gyBTxEnvCollateral     :: !GYUTxO
     }
 
 utxoFromTxInDetailed :: GYTxInDetailed v -> GYUTxO
 utxoFromTxInDetailed (GYTxInDetailed (GYTxIn ref _witns) addr val d ms) = GYUTxO ref addr val d ms
 
-data BuildTxException
```

Comment:
This was moved to `GeniusYield.Transaction.Common` so it could be imported in another module (`GeniusYield.TxBuilder.Errors`) without cyclic dep.